src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
```

Comment:
The `<form>` tag is self-closing, which is not valid in HTML. Forms should have an opening `<form>` tag and a closing `</form>` tag. Consider changing `<form />` to `<form>` and ensure it is properly closed with `</form>`.